### DISCUSSION TITLE
[Help]: Fade example sandbox does not open

### DISCUSSION THREAD

**Discussion by douglasjordan2** [davidjerleke/embla-carousel#741] [Help] [answered]
### Summary

When fade animation is brought up, [this comment](https://github.com/davidjerleke/embla-carousel/issues/39#issuecomment-568881867) is linked, where [this sandbox](https://codesandbox.io/s/embla-carousel-github-issue-39-sdkul) is linked. That sandbox does not load. 

Does anyone have the CSS that was in that sandbox?

**Comment by davidjerleke** [comment, marked as answer]
@douglasjordan2 unfortunately, CodeSandbox introduced their 20 sandbox limit, effective from January 30th this year. As a consequence, I had to delete a lot of sandboxes.

- You’ll have to wait for this: #39.

**Reply by douglasjordan2** [reply]
all good, m8, i figured out soon after that a pure slider wasn't going to fit what i needed so i just went another direction. looking forward to the fade plugin whenever it gets implemented.

no need to reply if you can't, but is `is-selected` deprecated? doesn't seem to work on my sliders. i have the ClassNames plugin but that just applied `is-snapped` and `is-in-view`. all good either way, i appreciate the reply.

**Reply by davidjerleke** [reply]
> is `is-selected` deprecated? doesn't seem to work on my sliders. i have the ClassNames plugin but that just applied `is-snapped` and `is-in-veiw`

@douglasjordan2 the `is-selected` class name has been replaced with `is-snapped`. But you can choose any class name you want using the [`snapped option`](https://www.embla-carousel.com/plugins/class-names/#snapped):

```ts
ClassNames({
  snapped: 'is-selected'
})
```

Cheers!
David
